app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -243,6 +245,13 @@ class PasswordFragment : Fragment(R.layout.password_recycler_view) {
         try {
             listener = object : OnFragmentInteractionListener {
                 override fun onFragmentInteraction(item: PasswordItem) {
+                    if (settings.getString(PreferenceKeys.SORT_ORDER) == PasswordRepository.PasswordSortOrder.RECENTLY_USED.name) {
+                        //save the time when password was used
```

Comment:
@msfjarvis Would it be better to always save the timestamp? I'm torn.

Reply:
Reading is quicker than writing so it's probably best to avoid a SharedPrefs edit when possible.

Reply:
Well, if we always save the timestamp, when this option is selected then everything will be sorted (and so probably  passwords won't be were they used to be... and could be a little weird to find at first), so i thought to keep an Independent sort at first and then start keeping track of the recent passwords.

Reply:
Agreed, let's keep this as is.